main.py

```diff
@@ -39,18 +40,36 @@ def write_pages(filename, pages):
         pdf_writer.write(output_file)
 
 
-# TODO: Take input and output file paths as flags,
+def parse_arguments():
+    parser = argparse.ArgumentParser(prog="Book Maker",
+                                     description="Converts PDF into books for easy printing")
+    parser.add_argument("input_pdf",
+                        type=str,
+                        help="Input PDF filepath"
+                        )
+    parser.add_argument("-o",
+                        "--output_pdf",
+                        type=str,
+                        help="Output PDF filepath",
+                        default="output.pdf"
```

Comment:
```suggestion
                        default="output.pdf",
```
Nit: Usually it's good to end with a comma, so that when you add a new param in the next line, the changes shows +1 lines changes as opposed to +2.

Will raise a separate issue to switch to a formatter that auto handles this. For now, manually add commas wherever necessary.